Version 2.5.0 (#630)
BlueBrain/HighFive: CHANGELOG.md

## Version 2.5.0 - 2022-11-03
### New Features
    - Enable collective MPI IO thought Data Transfer Property (#623). Thanks to Rob Latham.
    - Add a support for half-precision (16-bit) floating-point based on the Half library (http://half.sourceforge.net) (#587). Thanks to Sergio Botelh.
    - Add an AllocationTime property for DataSet Create Props (#627).

>     - Add possibility to get and set file space strategy (#618).

Reviewer: ```suggestion
    - Add possibility to get and set file space strategy (#618). For page allocated files wrap the API to set/retrieve the page size.
```

>     - Add API for getting Access and Create property lists from differents kind of objects (#629).

Reviewer 2: ```suggestion
    - Add API for getting Access and Create property lists from different kinds of objects (#629).
```

>     - Add API for getting Access and Create property lists from different kinds of objects (#629).

Reviewer: ```suggestion
    - Add API for getting Access and Create property lists of HighFive objects (#629).
```

>     - Let user configure metadata reads and writes at file level (#624).

Reviewer: ```suggestion
    - Let users configure metadata reads and writes at file level (#624). Thanks to Rob Latham.
```

> ### Improvments

Reviewer 2: ```suggestion
### Improvements
```

>     - MPIOFileDriver is now deprecated. Use FileAccessprops (#622).

Reviewer 2: ```suggestion
    - MPIOFileDriver is now deprecated. Use FileAccessProps (#622).
```

    - Support of block argument in API (#584).
    - Serialization of types is now automagic and so recursive (#586).

>     - Add an argument to specific File Access Properties in File class construtor (#626).

Reviewer: ```suggestion
    - Add an argument to specific File Create Properties in File class construtor (#626).
```
